be/src/exec/vectorized/analytic_node.cpp

```diff
@@ -335,6 +335,9 @@ Status AnalyticNode::_fetch_next_chunk(RuntimeState* state) {
             ColumnPtr column = _analytor->order_ctxs()[i]->evaluate(child_chunk.get());
             _analytor->append_column(chunk_size, _analytor->order_columns()[i].get(), column);
         }
+        RETURN_IF_ERROR(state->check_mem_limit("AnalyticNode"));
+    } catch (std::bad_alloc const&) {
+        return Status::MemoryLimitExceeded("Mem usage has exceed the limit of BE");
```

Comment:
It is recommended to give a special logo to indicate that it is caused by OOM to distinguish between ordinary memory exceeding the limit

Reply:
if catch exception will output "Mem usage has exceed the limit of BE" , others will output Specific error message
e.g.:             str << "Mem usage has exceed the limit of single query, You can change the limit by "
                   "set session variable exec_mem_limit.";

Reply:
The output of memory-related error information will be unified in the future